docs/content/contribution-process.md

```diff
@@ -1,14 +1,17 @@
 ---
 title: "Contribution process"
-weight: 50
+weight: 9
 aliases:
   - /docs/getting-started/contributing
   - /getting-started/contributing
   - /get-started/contributing
+  - /docs/get-started/contribution-process
```

Comment:
A long time ago there was a `/docs` folder _inside_ `docs/content`, which is where the first alias is from. You don't need it here.

```suggestion
  - /get-started/contribution-process
```